lmfdb/ecnf/WebEllipticCurve.py

```diff
@@ -249,8 +261,7 @@ def make_E(self):
             ('Label' , self.label)]
         
         # Plot
-        n1 = len(E.base_field().embeddings(RR))
-        if(n1):
+        if(E.base_field().signature()[0]):
```

Comment:
FIxed :) Now guess which is the programming language I learnt when I was in high school...